README.md

```diff
@@ -9,7 +9,7 @@
 
 ## How to use
 
-1. Download and copy this repository into the `plugins` directory of the Camunda Modeler
+1. Download and copy this repository into the `plugins` directory of the Camunda Modeler. Directory should look like plugins/plugin/
```

Comment:
What shall `plugin` refer to? The name of the plug-in? In this case, please add a placeholder at least, i.e. Each plug-in shall live in its own sub-directory, i.e. under `plugins/{PLUGIN_NAME}`.

Alternatively, consider _"Copy this repository as a directory into the `plugins` directory..."_